pkgs/applications/graphics/sane/backends/brscan4/default.nix

```diff
@@ -88,6 +88,9 @@ stdenv.mkDerivation rec {
   meta = {
     description = "Brother brscan4 sane backend driver";
     homepage = "http://www.brother.com";
+    # if we don't specify this, a default will be grapped from src,
+    # which doesn't eval on all platforms.
+    repository = [ ];
```

Comment:
> the eval for unsupported platforms shouldn't fail anymore.

i'm not sure what makes you think that? brscan4 still has a src that fails to eval on unsupported platforms..

i'm still going to change this, because i think making src eval on all platforms is a cleaner fix.

Reply:
> > the eval for unsupported platforms shouldn't fail anymore.
> 
> i'm not sure what makes you think that? brscan4 still has a src that fails to eval on unsupported platforms..

```
nix-instantiate -A brscan4 --show-trace --argstr system aarch64-linux
```

Reply:
hmm, i see, i was actually testing with `nix eval .#pkgsCross.loongarch64-linux.brscan4.meta.repository`, which produces an error.  i suppose i can remove this altogether, hopefully it doesn't cause another hydra build failure down the line.